sql/src/main/java/io/crate/analyze/MetaDataToASTNodeResolver.java

```diff
@@ -98,6 +98,10 @@ public Extractor(DocTableInfo tableInfo) {
             if (pk != null) elements.add(pk);
             // index definitions
             elements.addAll(extractIndexDefinitions());
+            // check constraints
+            if (!tableInfo.checkConstraints().isEmpty()) {
+                elements.addAll(tableInfo.checkConstraints());
+            }
```

Comment:
```suggestion
            elements.addAll(tableInfo.checkConstraints());
```

The `// check constraints` comment doesn't add any information. `addAll` also behaves the right way if the collection is empty.